test/integration/targets/setup_mysql_db/handlers/main.yml

```diff
@@ -0,0 +1,25 @@
+- name: stop mysql service
+  service:
+    name: "{{ mysql_service }}"
+    state: stopped
+  listen: cleanup mysql
+
+- name: remove mysql packages
+  action: '{{ ansible_pkg_mgr }}'
+  args:
+    name: "{{ item }}"
+    state: absent
+  loop: "{{ mysql_packages | union(mysql_cleanup_packages) }}"
+  listen: cleanup mysql
+
+- name: remove mysql data
+  file:
+    path: "{{ item }}"
+    state: absent
+  loop: "{{ mysql_data_dirs }}"
+  listen: cleanup mysql
+
+- name: remove pip packages
+  pip:
+    name: mysql-python
+    state: absent
```

Comment:
```suggestion
    state: absent
    listen:  remove pip packages
```

Reply:
I don't think this needs a `listen` statement since it's called referenced by name directly and we don't need any other handlers trigger for FreeBSD cleanup.